File: Data Structures & Algorithms/concatenation-of-array/submission-1.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        length = len(nums)

        newArr = [0] * length * 2

        for i in range(length):
            newArr[i] = newArr[i + length] = nums[i]

        return newArr

File: Data Structures & Algorithms/duplicate-integer/submission-1.go
func hasDuplicate(nums []int) bool {
    unique := make(map[int]struct{})

    for _, num := range nums {
        _, ok := unique[num]
        if ok {
            return true 
        } else {
            unique[num] = struct{}{}
        }
    }

    return false
}
